tests/test_network.py

```diff
@@ -1494,7 +1494,43 @@ def test_dump_to_string_with_report():
     report2 = str(reporter)
     assert len(report2) >= len(report1)
 
-
+def test_attach_line_on_line():
+    n = pp.network.create_eurostag_tutorial_example1_network()
```

Comment:
This test is confusing and must be improved. The voltage_level_id and the bbs_or_bus_id should be created in the test, just after creating the network. This new voltage level will have a new load or a new generator.